### PR TITLE
Updates symbol serialization in all Location subclasses in Scanner

### PR DESCRIPTION
**Pull request by nimakarimipour**
Followup to #213, this PR fixes symbol serialization in all `Location` subclasses.

### CODE REVIEW

File: annotator-scanner/src/main/java/edu/ucr/cs/riple/scanner/location/MethodLocation.java [hunk 1]
```diff
@@ -45,7 +46,7 @@ public String tabSeparatedToString() {
         "\t",
         type.toString(),
         enclosingClass.flatName(),
```
Comment (msridhar) [inline annotator-scanner/src/main/java/edu/ucr/cs/riple/scanner/location/MethodLocation.java:48]: Should we do this, for consistency?
```suggestion
        Serializer.serializeSymbol(enclosingClass),
```

Comment (nimakarimipour) [reply annotator-scanner/src/main/java/edu/ucr/cs/riple/scanner/location/MethodLocation.java:48]: Sure, we can. I will update it for all locations.

Comment (nimakarimipour) [reply annotator-scanner/src/main/java/edu/ucr/cs/riple/scanner/location/MethodLocation.java]: @msridhar Updated in all subclasses, looks much better now.

**Review by msridhar — APPROVED** [review]
👍

**Pull request merged**